target-platform/target-platform.target

```diff
@@ -106,17 +108,32 @@
 				</dependency>
 			</dependencies>
 		</location>
-		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="Maven-archetype" missingManifest="generate" type="Maven">
+		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="Maven-archetype Common" missingManifest="generate" type="Maven">
 			<dependencies>
 				<dependency>
 					<groupId>org.apache.maven.archetype</groupId>
-					<artifactId>archetype-catalog</artifactId>
+					<artifactId>archetype-common</artifactId>
 					<version>3.2.1</version>
 					<type>jar</type>
 				</dependency>
+			</dependencies>
+			<instructions><![CDATA[
+Bundle-Name:           M2Eclipse supplement fragment for ${mvnGroupId}:${mvnArtifactId}:${mvnVersion}
```

Comment:
This is not a fragment.

Reply:
Removed fragment. But I think something like 
`M2Eclipse Archetype supplement based on ${mvnGroupId}:${mvnArtifactId}:${mvnVersion}`
would be better here and below.

Reply:
That is the artifactId. That's fine, but I would also like to mention in the name too that this bundle is wrapped, so I think simply `M2Eclipse's ${mvnArtifactId}` is good.